service/src/main/java/uk/gov/hmcts/reform/fpl/events/CMOEvent.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.fpl.events;
+
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+
+@Value
+@EqualsAndHashCode(callSuper = true)
+public class CMOEvent extends CallbackEvent {
```

Comment:
Would rename to `CaseManagementOrderIssuedEvent` or something like that, CMOEvent isn't very clear as to when this event should be used